bentoml/_internal/models/store.py

```diff
@@ -61,7 +61,7 @@ def _process_model_tag(tag: str) -> t.Tuple[str, str]:
 class StoreCtx(object):
     name = attr.ib(type=str)
     version = attr.ib(type=str)
-    path = attr.ib(type=PathType)
+    path = attr.ib(type=str)
```

Comment:
it seems to me there's no real need for the PathType, should we remove that?

Reply:
I think it's still valuable to allow `PathType` in user-facing APIs, but we can normalize to string internally.